frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/storage-review.tsx

```diff
@@ -74,32 +83,41 @@ const StorageReviewFirehose: React.FC<StorageReviewFirehoseProps> = ({
     (sc) => getAnnotations(sc, {})[DEFAULT_SC_ANNOTATION] === 'true',
   );
 
+  const rows = combinedDisks.map(
+    (combinedDisk ) => {
+      const size = combinedDisk.getSize();
+      return [
+        combinedDisk.getName(),
+        combinedDisk.getDiskInterface(),
+        size && `${size?.value} ${size?.unit}`,
+        combinedDisk.getStorageClassName(),
+        combinedDisk.getAccessModes()?.[0].toString(),
```

Comment:
can you join this array in case there are multiple of them? - but shouldn't be

Reply:
Done!